cdc/owner/feed_state_manager.go

```diff
@@ -77,16 +78,16 @@ func newFeedStateManager(up *upstream.Upstream) *feedStateManager {
 	// backoff will stop once the defaultBackoffMaxElapsedTime has elapsed.
 	f.errBackoff.MaxElapsedTime = defaultBackoffMaxElapsedTime
 
-	f.resetErrBackoff()
-	f.lastErrorTime = time.Unix(0, 0)
-
+	f.resetErrRetry()
+	f.lastErrorRetryTime = time.Unix(0, 0)
```

Comment:
```suggestion
	f.lastErrorRetryTime = time.Unix(0, 0)
```
This is redundant since lastErrorRetryTime is initialized in `f.resetErrRetry()`.